popup.css

```diff
@@ -0,0 +1,116 @@
+.model {
```

Comment:
- [ ] Great job styling your modal popup it looks really good. However, kindly note that you are supposed to stick as much as possible to the chosen Figma design. Currently, your modal popup for desktop is fixed at the bottom side of the screen and it is supposed to be placed on the middle side of the screen. Kindly fix this.

**This is how your design currently loos like:**

![image](https://user-images.githubusercontent.com/86318284/180270409-316897c3-e920-46a5-a268-b141a5d99acd.png)

**This is the desired Figma design:**

![image](https://user-images.githubusercontent.com/86318284/180270589-82171c15-4aa1-4205-9e5d-b9191faa0648.png)